data/mil2525d/utilities/military-features-utilities/README.md

```diff
@@ -18,6 +20,66 @@
 
 ## Instructions
 
+### Create Military Features Geodatabase Template
+
+#### Overview
+
+The utility creates a new File Geodatabase (GDB) in a specified folder, replacing any existing geodatabase of the same name in that folder.  The version of the newly created geodatabase can also be specified.
+
+The structure of the geodatabase is determined by the contents of the CSV files found in a  folder selected at run-time.  The most recent version of these CSV files can be found [here](https://github.com/Esri/joint-military-symbology-xml/tree/master/samples/military_feature_schemas).
+
+A Versions table is added to the newly created geodatabase.  The contents of that Versions table is based on the [versions.csv](https://github.com/Esri/military-features-data/blob/master/data/mil2525d/utilities/style-utilities/merge-stylx-utilities/versions.csv) file found in this repo.  The "Version" information for `automated_creation_date` and `last_modification_date` are populated with the current date when the tool is run.
+
+Each feature class created in the geodatabase is populated with the field specifications found in the corresponding field schema CSV files, also found [here](https://github.com/Esri/joint-military-symbology-xml/tree/master/samples/military_feature_schemas) and documented [here](https://github.com/Esri/joint-military-symbology-xml/blob/master/samples/README.md).
+
+#### Steps
+
+* Obtain the latest set of Military Features source data and utilities
+	* Clone/download this repository to your local machine.
+* Obtain the latest set of `military_feature_schemas` source CSV files
+	* Clone/download the [Joint Military Symbology XML (JMSML) repository](https://github.com/Esri/joint-military-symbology-xml).
+* Run ArcGIS Pro
+* Navigate to the local location of the [template-gdb-toolbox GeoProcessing Toolbox](./template-gdb-toolbox).
+* Run the *Create Military Features Geodatasbase Template* tool, found in the `GeodatabaseTemplate` toolbox.
+	* For the `Schemas Folder` parameter, specify a folder containing the aforementioned schema CSV files.  These are normally found in your local JMSML samples folder, the clone you made of [this](https://github.com/Esri/joint-military-symbology-xml/tree/master/samples/military_feature_schemas).
+	* For the `Geodatabase Destination Folder` parameter, specify a destination folder for the newly created file geodatabase.  The name used to create the geodatabase, together with its entire dataset structure, is specified in the `Schemas.csv` file found in the schemas folder.  Any existing geodatabase in the output folder, with the same name, will be overwritten.
+	* For the `Version` parameter, select from the list of possible geodatabase versions.  ArcGIS 10.0 is the default version.
```

Comment:
Oops (slightly my bad)  - Looks like **"Current"** is ArcGIS Desktop **10.1** through **Pro** - so can you change that back to "Current" (from your previous description here it sounded like this only worked on Pro - but check this and it works 10.1-Pro.